bloom-housing/ui-seeds
# feat: drawer component

## Issue Overview

This PR addresses #25

## Description

This PR adds the Drawer component.

## How Can This Be Tested/Reviewed?

Check out the stories and ensure the component matches the Figma designs.

Some thoughts / remaining questions
* The size used for the close icon was not a size we already supported. I added one additional icon size of `2xl` and shifted some of the other sizes in response. This could result in breaking the current icon uptake since two of the names did change size.
* Even though the header text is using the right size from the Figma it looks too large, and it seems like the Figma is using a different font weight. Ideally the heading styles in the Drawer match 1-1 with our existing typography.
* The padding in the footer looks a little off despite matching the Figma. To me it seems like the padding in between buttons should maybe be smaller than the content padding?
* The fact that the x icon sits a little to the left of the content seems a little off despite matching the Figma.

## Checklist:
